lib/manager/actions/status.js

```diff
@@ -134,6 +136,51 @@ export async function fetchAppInfo () {
   }
 }
 
+function getMergeFeedModalContent (result: MergeFeedsResult): ModalContent {
+  const details = []
+  // Do nothing or show merged feed modal? Feed version is be created
+  details.push('Remapped ID count: ' + result.remappedReferences)
+  if (Object.keys(result.remappedIds).length > 0) {
+    const remappedIdStrings = []
+    for (let key in result.remappedIds) {
+      // Modify key to remove feed name.
+      const split = key.split(':')
+      const tableAndId = split.splice(1, 1)
+      remappedIdStrings.push(`${tableAndId.join(':')} -> ${result.remappedIds[key]}`)
+    }
+    details.push('Remapped IDs: ' + remappedIdStrings.join(', '))
+  }
+  if (result.skippedIds.length > 0) {
+    const skippedRecordsForTables = {}
+    result.skippedIds.forEach(id => {
+      const table = id.split(':')[0]
+      if (skippedRecordsForTables[table]) {
```

Comment:
Suggestion: 
`const skippedRecords = skippedRecordsForTables[table]
skippedRecordsForTables[table] = skippedRecords ? 1 : skippedRecords + 1 
`

Reply:
I think your ternary is backwards?

Reply:
Probably 🤔

Reply:
How about:
```
skippedRecordsForTables[table] = (skippedRecordsForTables[table] || 0) + 1
```
That doesn't seem too clever, but is nice and concise.

Reply:
That works too.